overrides/default.nix

```diff
@@ -137,9 +137,21 @@ lib.composeManyExtensions [
             self.dopy
             self.ncclient
           ];
-        } // lib.optionalAttrs (lib.versionOlder old.version "5.0") {
-          prePatch = pkgs.python.pkgs.ansible.prePatch or "";
-          postInstall = pkgs.python.pkgs.ansible.postInstall or "";
```

Comment:
Note that `prePatch` and `postInstall` have been empty in nixpkgs for a long time :). I looked back in time and added the patches to the right package `ansible-base`.